### DISCUSSION TITLE
「ストック」ボタンについての疑問

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#702] [要望・アイデア・不具合]
①　自分の記事内で「いいね」マークにマウスオーバーすると「立ち入り禁止」表示が出ます。しかし、「ストック」マークでは出ません。ストック操作も可能で、試しにクリックしたところ Contributions も一つ増えました。投稿者自身がクリックして Contributions を増やせるというのは如何なものでしょうか？

②　マイページの投稿記事リストの各記事の右側には「ストック」ボタンがあります。普通は自分の記事をストックする必要は無いのに、何のためのボタンですか？　その一方、その記事がストックされた件数の表示はありません。「ストック」ボタンは、「ストック件数」表示のつもりでの誤実装ではありませんか？

「ストック」ボタンを「ストック件数」表示に訂正するのであれば、ついでに「コメント数」表示の追加もお願いします。今回、記事本文の末尾に「コメント数」表示が追加されたのですから、そのほうがバランスがとれます。

**Comment by tanaka-asami415** [comment]
@tsubolabo 
お問合せありがとうございます。

ストックはご自身の記事も分類等、管理しておきたいシーンを想定して、ストックできる仕様となっております🙇
自分の記事に対するストックは、今後contributionを加算しないようにする事を検討いたします。

**Reply by tsubolabo** [reply]
@tanaka-asami415 
> ストックはご自身の記事も分類等、管理しておきたいシーンを想定して、ストックできる仕様となっております

　ストックの限られた機能から見ても、そのようなシーンは具体例としてなかなか思い浮かんできませんが、それは私の想像,推察力（≒妄想,邪推気質）が足りないせいだということにしても、疑問②についてはどうでしょうか？

　マイぺージにおいて、いちいち記事を開かずに、自分の記事への いいね数、ストック数、コメント数 を知りたいというニーズは、自分の記事をストックしたいというニーズよりも遥かに多いと思います。今のところは、いいね数 しか分かりません。

**Reply by tanaka-asami415** [reply]
@tsubolabo 
> マイぺージにおいて、いちいち記事を開かずに、自分の記事への いいね数、ストック数、コメント数 を知りたいというニーズ

ご意見ありがとうございます。
自分自身の記事の数値を簡単に確認したいというニーズはありそうですので、参考にさせていただきます。

現時点の代替案としては、ひと手間必要ですが、検索オプション `user:` でご自身を指定して検索いただくと、一覧で確認していただくことが可能ですので、よろしければご活用ください🙇
